AllReadyApp/Web-App/AllReady/web.config

```diff
@@ -1,9 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <system.webServer>
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
     </handlers>
-    <aspNetCore processPath=".\AllReady.exe" arguments="" forwardWindowsAuthToken="false" stdoutLogEnabled="false" />
+    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" forwardWindowsAuthToken="false" stdoutLogEnabled="false" />
```

Comment:
@stevejgordon, yeah, this is a mystery to me. When I run locally, VS 2015 changes this to `.\AllReady.exe`, from the original value of `%LAUNCHER_PATH%`. I'm not too sure why. I've been good an undoing my changes to this file before commiting, but this time I forgot. Seeing how this PR built successfully in AppVeyor, my assumption is it builds fine. As far as it actually running in whatever environment we have set up that is not our local dev environment, I'm not too sure.